pkg/mqtrigger/messageQueue/messageQueue.go

```diff
@@ -44,6 +44,8 @@ type (
 	MessageQueueConfig struct {
 		MQType string
 		Url    string
+		// Secrets *v1.Secret
```

Comment:
What is this for? Maybe we can remove it?

Reply:
Removed